pkg/prometheus/promcfg_test.go

```diff
@@ -1322,6 +1321,69 @@ func TestAlertmanagerRelabelConfigs(t *testing.T) {
 	})
 }
 
+func TestAlertmanagerAlertRelabelConfigs(t *testing.T) {
+	for _, tc := range []struct {
+		name    string
+		version string
+		golden  string
+	}{
+		{
+			name:    "Invalid Prometheus Version",
+			version: "2.40.1",
+			golden:  "AlertmangerAlertRelabel_Invalid_Version.golden",
+		},
+		{
+			name:    "Valid Prometheus Version",
+			version: "2.51.0",
+			golden:  "AlertmangerAlertRelabel_Valid_Version.golden",
+		},
+	} {
+		t.Run(fmt.Sprintf("AlertmanagerAlertRelabelConfigs: %s", tc.name), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(tc.name, func(t *testing.T) {
```

nitpicking: when running tests we already have the test name in the output, so there's no need to repeat it here